doc/man/scons.xml

```diff
@@ -4237,6 +4237,13 @@ Calling &Variables; with no arguments is equivalent to:
 vars = Variables(files=None, args=ARGUMENTS)
 </programlisting>
 
+<para>
+Note that since the variables are eventually added as &consvars;,
+you should choose variable names which do not conflict with
```

Comment:
maybe uninintentionally change predefined consvars.. instead of conflict?

Reply:
Sure, that sounds better.